### PR TITLE
feat: Add ParenthesisValidation class

### PR DESCRIPTION
**Pull request by RogerMo01**

### CODE REVIEW

File: OOP-Exercises/ParenthesisValidation.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOP_Exercises
+{
+    class ParenthesisValidation
```
Comment (LongJohnSilver1504) [inline OOP-Exercises/ParenthesisValidation.cs:9]: Good but  assume the string only has opened and closed parenthesis.

**Review by LongJohnSilver1504 — CHANGES_REQUESTED** [review]
Good but why ot static static class

File: OOP-Exercises/ParenthesisValidation.cs (hunk 1 above)
Comment (RogerMo01) [reply OOP-Exercises/ParenthesisValidation.cs]: Done!

**Review by LongJohnSilver1504 — APPROVED** [review]
Good but remember commit convention:
instead:
Assume the string only has opened and closed pharentheses
fix: String only has opened and closed pharentheses
LGTM!

**Pull request merged**